spec/suites/map/MapSpec.js

```diff
@@ -2201,6 +2201,39 @@ describe('Map', () => {
 		});
 	});
 
+	describe('#project', () => {
+
+		it('throws if map is not set before', () => {
+			expect(() => {
+				map.panInsideBounds();
+			}).to.throw();
+		});
+
+		it('returns pixel coordinates relative to the top-left of the CRS extents', () => {
+			map.setView([40, -83], 5);
+			const x = latLng([40, -83]);
+			const a = map.project(x, 5);
+			expect(a.x).to.be.equal(2207.2888888888892);
+			expect(a.y).to.be.equal(3101.3204602149353);
+		});
+
+		it('test the other corridinates', () => {
+			map.setView([40, 83], 5);
+			const x = latLng([40, 83]);
+			const b = map.project(x, 5);
+			expect(b.x).to.be.equal(5984.711111111111);
+			expect(b.y).to.be.equal(3101.3204602149353);
```

Comment:
Floating point number literals with lots of decimals are scary in tests, since they can hit rounding errors.

Instead of `equal`, use https://www.chaijs.com/api/assert/#method_approximately with a delta of... 1/1000000, maybe?